SygicMapsKit/Classes/SearchModule/SYMKSearchModel.swift

```diff
@@ -33,9 +33,7 @@ class SYMKSearchModel {
     /// Max number of results search returns.
     public var maxResultsCount: UInt = 10
     
-    // MARK: - Private Properties
-    
-    private let search = SYSearch()
+    public var search: SYSearch?
```

Comment:
toto nema moc zmysel mat public. skor by som to nechal private a cely SYMKSearchModel inicializoval az po SDK doinicializovani